### PR TITLE
Mdprescott/add started at

### PR DESCRIPTION
**Pull request by mdprescott**
Adding a started_at column to the Users table. This way we can track/add/filter/display the User started_at. (when an employee started with the company)

### CODE REVIEW

File: db/migrate/20181105223440_add_started_to_users.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class AddStartedToUsers < ActiveRecord::Migration[5.2]
+  def change
+    add_column :users, :started_at, :date
```
Comment (jerflan) [inline db/migrate/20181105223440_add_started_to_users.rb]: Why are you including this in the same PR as the user's controller? They should be separate.

Comment (jerflan) [reply db/migrate/20181105223440_add_started_to_users.rb]: You also might consider having an `up` and `down` method so you can rollback the db to easily remove the column if need be.

Comment (mdprescott) [reply db/migrate/20181105223440_add_started_to_users.rb]: gah - forgot to revert my local before opening a new pr

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -0,0 +1,75 @@
+class UsersController < ApplicationController
+  before_action :set_user, only: [:show, :edit, :update, :destroy]
+
+  # GET /users
```
Comment (jerflan) [inline app/controllers/users_controller.rb]: These comments are noisy.

**Pull request merged**